eisbuk-admin/src/components/atoms/UserAttendance/UserAttendance.tsx

```diff
@@ -35,26 +50,60 @@ const UserAttendance: React.FC<Props> = ({ attended, userBooking, slotId }) => {
       })
     );
   };
+  const handleSelect = (interval: string) => {
+    dispatch(
+      selectInterval({
+        slotId,
+        userId: userBooking.customer_id,
+        interval,
+      })
+    );
+  };
 
   const absenteeButtons = (
-    <Button
-      data-testid={`${userBooking.name}${userBooking.surname}`}
-      variant="contained"
-      size="small"
-      color={attended ? "primary" : "secondary"}
-      onClick={handleClick}
-      disabled={localAttended !== attended}
-    >
-      {attended ? "👎" : "👍"}
-    </Button>
+    <FormControl className={classes.formControl}>
+      <InputLabel>Intervals</InputLabel>
+      <Select
+        data-testid={`${userBooking.name}-${userBooking.surname}-select`}
+        value="13:00 - 14:00"
+        onChange={() => handleSelect("13:15 - 14:15")}
+        // materialUI <select> element is an abstraction for a bunch of other elements
+        // inputProps sets the testid of <input> element inside the <select>
+        // so that rtl can fireEvents on it
+        inputProps={{
+          "data-testid": `${userBooking.name}-${userBooking.surname}-input`,
+          disabled: !localAttended,
+        }}
+      >
+        {intervals.map((interval) => (
+          <option
+            key={`${userBooking.name}${userBooking.surname}${interval}`}
+            value={interval}
+          >
+            {interval}
+          </option>
+        ))}
+      </Select>
+      <Button
+        data-testid={`${userBooking.name}-${userBooking.surname}`}
+        variant="contained"
+        size="small"
+        color={attended ? "primary" : "secondary"}
+        onClick={handleClick}
+        disabled={localAttended !== attended}
+      >
+        {attended ? "👎" : "👍"}
+      </Button>
+    </FormControl>
   );
+
   return (
     <ListItem className={listItemClass}>
       <ListItemAvatar>
         <EisbukAvatar {...userBooking} />
       </ListItemAvatar>
       <ListItemText primary={userBooking.name} />
-      <ListItemSecondaryAction>{absenteeButtons}</ListItemSecondaryAction>
+      <ListItemSecondaryAction>{absenteeButtons}</ListItemSecondaryAction>;
```

Comment:
This will render `;` in the document

Reply:
this has been screaming warnings at me and I couldn't spot it at all...